app/models/Comment.ts

```diff
@@ -84,6 +86,12 @@ class Comment extends Model {
    */
   resolvedById: string | null;
 
+  /**
+   * Active reactions for this comment.
+   */
+  @observable
+  reactions: Reaction[];
```

Comment:
Lets clarify that these are not full `Reaction` objects with a different type, `ReactionDetail` or something like that